docs/mkdocs.yml

```diff
@@ -23,31 +23,53 @@ site_dir: 'generated'
 
 nav:
   - Welcome:
-      - Landing: index.md 
-  - Overview:
-      - Readme: readme.md
-      - User guide: direct/user-guide.md
+      - Landing: index.md
+  - What is KubeStellar?:
+      - Overview: readme.md
       - Architecture: direct/architecture.md
+      - Related:
+        - KubeFlex: direct/kubeflex-intro.md
+        - KubeStellar Galaxy: direct/galaxy-intro.md
       - Release-notes: direct/release-notes.md
-  - Getting Started: 
-          - Pre-reqs: direct/pre-reqs.md
-          - Examples: direct/examples.md
-          - Using existing hosting cluster: direct/hosting-cluster.md
-          - KubeStellar on K3D: direct/deploy-on-k3d.md
+  - Quick Start: direct/get-started.md
+  - User Guide:
+          - Basics: direct/usage-basics.md
+          - Prerequisites: direct/pre-reqs.md
+          - Quick Start: direct/get-started.md
+          - Setting up KubeStellar:
+              - Set Up Kubestellar Core Components: 
+                - Try the "Common-setup": direct/common-setup-intro.md
+                - Step by Step: direct/common-setup-step-by-step.md
+                - Via Helm Chart: direct/common-setup-core-chart.md
```

Comment:
These are still written as if they are part of the quickstart rather than being general instructions. Not sure which you intend.

Reply:
See my comment about building the quickstart by using includes. More editing of the associated files is definitely required.